docs/user_manual/serialization.md

```diff
@@ -0,0 +1,159 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# Serialization
+
+The power-grid-model provides tools for serialization and deserialization for datasets.
+
+Visit the [Python API Reference](../api_reference/python-api-reference.md#utils) and [C API Reference](../api_reference/power-grid-model-c-api-reference.rst) for the full documentation of the serialization tools.
```

Comment:
Do we already need to refer to the new serialization workshop in https://github.com/PowerGridModel/power-grid-model-workshop/pull/17 ? That link would not work and cannot be tested until that PR is merged

Reply:
We do this in another PR for all the examples